hepcrawl/loaders.py

```diff
@@ -165,5 +164,40 @@ class HEPLoader(ItemLoader):
     )
     urls_out = ListToValueDict()
 
+    def get_dois(self, dois=None, material=None):
```

Comment:
This should not be called `get_` as it's not getting anything, that's usually reserved for functions that return an element of what you want to get (a function called `get_name` is usually expected to return a `name`, whatever that would be, to set the name, you would use `set_name` instead). And usually those functions are not expected to change the object they are called on.

It should be better named like `add_dois`, to keep the style it uses when adding other fields. If that collides with any magic it might be doing internally, `set_dois` is also a good name.